shoop/core/locale/zh_Hans/LC_MESSAGES/django.po

```diff
@@ -747,6 +747,11 @@ msgstr "主分类"
 msgid "default price"
 msgstr "默认价格"
 
+#, fuzzy
+#| msgid "minimum purchase"
+msgid "minimum price"
+msgstr "最低购买"
```

Comment:
I made a separate issue out of this concern. https://github.com/shoopio/shoop/issues/326